common/reviews/api/node-library-build.api.ts

```diff
@@ -13,7 +13,7 @@ class ApiExtractorTask extends GulpTask<IApiExtractorTaskConfig> {
 
 class CleanTask extends GulpTask<void> {
   constructor();
-  executeTask(gulp: gulp.Gulp, completeCallback: (result?: Object) => void): void;
+  executeTask(gulp: gulp.Gulp, completeCallback: (result?: Object) => void): Promise<any>;
```

Comment:
Why would return a promise AND handle a completeCallback?  Isn't that redundant?